app/components/TableAdvancedSearchFilterMenu.tsx

```diff
@@ -53,7 +53,7 @@ export default function TableAdvancedSearchFilterMenu({
     <Row className="p-2 bg-light">
       {queryConfig.filterCards.map((filterCard) => {
         return (
-          <Col key={filterCard.id}>
+          <Col key={filterCard.id} xs={12} md={3}>
```

Comment:
keeps those filter cards from growing too wide

Reply:
ah yes—there's some questionable CSS working on this filter menu that i hoped you wouldn't notice. [here's](https://github.com/cityofaustin/vision-zero/pull/1632#issuecomment-2554832495) another short thread about this menu, too. i will open a follow-up issue to get this UI straightened out 🙏